File: LP/C/desafio7.c
#include<stdio.h>

void deslocamento_x(int *vetor, int x){
    
    for(int i = 0; i < 4; i++){
        vetor[i] = vetor[i] + x;
    }
}

int main(){

    int x;
    int vetor[4] = {1,2,3,4};
    printf("Insira o valor de x : ");
    scanf("%i", &x);
    
    deslocamento_x(vetor, x);

    for(int i = 0; i < 4; i++){
        printf("O %i valor do vetor novo eh: %i\n", i + 1, vetor[i]);
    }
    
}

File: LP/python/desafio7.py
vet = [1,2,3,4]

def deslocamento_x(x):
    for i in range(len(vet)):
        vet[i] = vet[i] + x

x = int(input("Insira o valor de x : "))
deslocamento_x(x)
print(vet)